bin/millau/node/src/service.rs

```diff
@@ -230,7 +231,7 @@ pub fn new_full(mut config: Configuration) -> Result<TaskManager, ServiceError>
 
 		let justification_stream = grandpa_link.justification_stream();
 		let shared_authority_set = grandpa_link.shared_authority_set().clone();
-		let shared_voter_state = sc_finality_grandpa::SharedVoterState::empty();
+		let shared_voter_state = shared_voter_state.clone();
```

Comment:
I guess we need the same fix for `rialto` node?

Reply:
Yeah, will do

Reply:
done